pkg/install/install.go

```diff
@@ -28,30 +29,37 @@ func (obj *installer) WithStatusWriter(sw statuswriter.StatusWriter) *installer
 
 // InstallBlocking creates a new v1alpha1.Package custom resource in the cluster and waits until
 // the package has either status Ready or Failed.
-func (obj *installer) InstallBlocking(ctx context.Context, pkg *v1alpha1.Package) (*client.PackageStatus, error) {
+func (obj *installer) InstallBlocking(ctx context.Context, pkg *v1alpha1.Package,
+	opts metav1.CreateOptions) (*client.PackageStatus, error) {
 	obj.status.Start()
 	defer obj.status.Stop()
-	pkg, err := obj.install(ctx, pkg)
+	if isDryRun(opts) {
+		_, err := obj.install(ctx, pkg, opts)
+		return nil, err
```

Comment:
This would lead to an error if err is nil. I think it would be best not to change the contract of this function, but you can also change the relevant behavior in `cmd/glasskube/cmd/install.go` so that `status == nil` is only considered an error if dry run has not been requested.